control/controlbase/conn.go

```diff
@@ -190,6 +216,14 @@ func (c *Conn) decryptOneLocked() error {
 		c.rx.next = 0
 	}
 
+	// Return out buffer to the pool if it's empty, lest we be
```

Comment:
typo: out > our? I think? Unless this was copy pasted from the tx path and you meant "the out buffer", but this isn't an out buffer.